Library/Homebrew/brew.sh

```diff
@@ -247,7 +247,11 @@ then
   # Hide shellcheck complaint:
   # shellcheck source=/dev/null
   source "$HOMEBREW_BASH_COMMAND"
-  { update-preinstall; "homebrew-$HOMEBREW_COMMAND" "$@"; exit $?; }
```

Comment:
I don't think this quite works the same as the previous version as Bash reads line-by-line and if this file is modified as part of the update process the new version may fail.